script.js

```diff
@@ -0,0 +1,44 @@
+const loadMovieInfo = (name) => {
```

Comment:
Our intent here was for you to make a network request to `/films/<film-id>` after grabbing the id of the selected movie. How would you have done it if you couldn't use your global variable `filmsObject`?. 

Think of this: You could store the film `id` in the `optionElement.value` so that when the user picks an option `selectElement.value` will have the id. [Check this](https://www.w3schools.com/jsref/prop_select_value.asp) click `Try it yourself`. Then make a second network request to `/films/<film-id>` to get the details of the movie.

Reply:
option.value = `<h3>${film.title}</h3><p>${film.release_date}</p><p>${film.description}</p>`
Then I would be able to set the innerHTML of the div to the value of the option.

Reply:
No, **`option.value = film.id`** then `option.innerText = film.title`.